### PR TITLE
sipgrep: new package

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: me / @diizzyy
Compile tested: ar71xx, mikrotik 750gl, r1122
Run tested: ar71xx, mikrotik 750gl, r1122

Description:
Add sipgrep to repo

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by diizzyy** [comment]
@hnyman

**Comment by diizzyy** [comment]
Also tested on Kirkwood arch

File: net/sipgrep/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+define Package/sipgrep/description
+  SipGrep is a utility for displaying and troubleshooting SIP signaling.
+endef
+
+CONFIGURE_ARGS += \
+	--enable-ncurses
+
+MAKE_FLAGS += \
+	CFLAGS="$(TARGET_CFLAGS)"
```
Comment (karlp) [inline net/sipgrep/Makefile]: as already mentioned, please drop this

Comment (diizzyy) [inline net/sipgrep/Makefile]: I'll re-submit later due to time constraints.

**Comment by diizzyy** [comment]
Fixed in #3200

**Pull request closed without merging**